.gitignore

```diff
@@ -16,3 +16,6 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+.ruby-env
```

Comment:
It might be nice to check these in, so everybody is developing with a consistent environment.